frontend/src/components/Breadcrumb.test.ts

```diff
@@ -0,0 +1,59 @@
+import { flushPromises, mount, shallowMount, VueWrapper } from '@vue/test-utils'
+import Breadcrumb from '@/components/Breadcrumb.vue'
+import { it, describe, expect, beforeEach } from 'vitest'
+import { routes } from '@/router'
+import { createRouter, createWebHistory, type Router } from 'vue-router'
+import { applicationsUserAdministers, selectedApplication } from '@/services/ApplicationState'
+import type { Application } from '@/services/ApplicationState'
+import type { Plugin } from 'vue'
+
+describe('Breadcrumb Component', () => {
+
+  let wrapper: VueWrapper;
+  let router: Router;
+  
+  beforeEach(async() => {
+    router = createRouter({
+      history: createWebHistory(),
+      routes: routes
+    })
+    wrapper = mount(Breadcrumb, {
+      global: {
+        plugins: [router]
+      }
+    })
+    await flushPromises()
```

Comment:
Not sure if we are 'await' anything in particular during 'beforeEach' setup?

Reply:
It is possible that mounting the component is asynchronous, and thus it is safest to wait until all promises are complete. This was required after calling router.push(), for example.